docs/security/authentication/active-directory/index.md

```diff
@@ -50,9 +53,32 @@ It is possible to use either `NTLM` or `Kerberos` authentication for Active Dire
 
 Without some additional configuration, AD authentication, whether forms-based or integrated, will usually fail on `kerberos` authentication and failback to `NTLM`.
 
+### Supported Setups for Active Directory Authentication {#ActiveDirectoryAuthentication-SupportedAuthentication}
+
+|                                 | Single Octopus Server | High-Availability |
+|---------------------------------|-----------------------|-------------------|
+| NTLM                            |         NTLM          |       NTLM        |
+| Negotiate                       |      Kerberos/NTLM    |       NTLM        |
+| IntegratedWindowsAuthentication |       Kerberos        |       NTLM        |
+
+:::hint
+ **Service Accounts and Kerberos**
+From Octopus version 2020.1.0 onwards, an upgrade to .Net Core 3.1 and usage of the HTTP.sys library, the Octopus Deploy Service running with Domain Service Account credentials, does not have ability to read the HttpContext.User.Identity.Name property which is used for kerberos authentication. There will be a requirement to run the Octopus Deploy Service as Local System in order to allow for kerberos to succesfully Authenticate. You can read more about this here: https://github.com/OctopusDeploy/Issues/issues/6602
+:::
+
 ### Configuring Kerberos Authentication for Active Directory {#ActiveDirectoryAuthentication-ConfiguringKerberos}
 
-The following configuration is required for Kerberos authentication:
+Here's a simple checklist to help you on your way to allowing Kerberos Authentication.
+
+1. Change Authentication Scheme 
+2. Set Octopus Deploy Bindings to use FQDN or NETbios name as per your usage
+3. Add the Octopus Deploy URL to the list of Trusted Sites
```

Comment:
> the list of Trusted Sites

Where is that? In each end users browser, or some other configuration area?

Reply:
All of these will have links to the appropriate sections (and perhaps links from the appropriate section back to the checklist) when I'm completely done with it.

The Trusted sites documentation so far: 
https://octopus.com/docs/security/authentication/active-directory#ActiveDirectoryAuthentication-AddingtrustedSitesviaGPO

Reply:
Ah Ok - nice - sounds like you have it handled :+1: